Content.Server/Resist/EscapeInventorySystem.cs

```diff
@@ -82,17 +88,26 @@ private void AttemptEscape(EntityUid user, EntityUid container, CanEscapeInvento
 
     private void OnEscape(EntityUid uid, CanEscapeInventoryComponent component, DoAfterEvent<EscapeInventoryEvent> args)
     {
-        if (args.Handled || args.Cancelled)
+        if (args.Cancelled)
+        {
+            component.CancelToken = null;
+            component.IsEscaping = false;
+            return;
+        }
+
+        if (args.Handled)
             return;
 
         Transform(uid).AttachParentToContainerOrGrid(EntityManager);
 
+        component.CancelToken = null;
+        component.IsEscaping = false;
         args.Handled = true;
     }
 
     private void OnDropped(EntityUid uid, CanEscapeInventoryComponent component, DroppedEvent args)
     {
-        //TODO: Enter cancel logic here
+        component.CancelToken = null;
```

Comment:
```suggestion
		component.CancelToken?.Cancel();
        component.CancelToken = null;
```

Reply:
Oop all done and the fix that prevented them from being cancelled